install-on-TPU-v4.sh

```diff
@@ -0,0 +1,24 @@
+sudo apt remove unattended-upgrades
+sudo apt update
+export PJRT_DEVICE=TPU
+export PATH="/home/artuskg/.local/bin:$PATH"
```

Comment:
I think you can generalize this, e.g.:
```suggestion
export PATH="$HOME/.local/bin:$PATH"
```